### DISCUSSION TITLE
A Shortlist of Python Library Requests

### DISCUSSION THREAD

**Discussion by zholmquist** [cloudflare/workerd#1946] [Python Packages]
First, rad, and thank you team! The initial list is solid, and looking forward to seeing how best to start bringing Workers into our pipeline ( along with other Cloudflare services ).

After y’all celebrate the launch, would love to see these on the board:

- Psycopg
- SQLModel
- SQLAlchemy
- Pendulum
- Python Jose

♥️ Thanks!

**Comment by URenko** [comment]
Additional, it would be better to be able to access Cloudflare D1 with some Python packages (e.g. SQLAlchemy) within Workers.

**Comment by jalexanderII** [comment]
SQLAlchemy would be a game changer!  🙏🏾

**Comment by cunhaac** [comment]
Waiting for SQLAlchemy

**Comment by stepheweffie** [comment]
python-multipart
nicegui

**Comment by shantanu561993** [comment]
Waiting for sqlalchemy

**Comment by superdarki** [comment]
SQLAlchemy, SQLModel and psycopg2 would be game changer

**Comment by TchiangW** [comment]
Expecting SqlAlchemy! Thank you.

**Comment by movaid7** [comment]
SqlAlchemy is supported through Pyodide
